include/zephyr/bluetooth/audio/audio.h

```diff
@@ -33,6 +33,35 @@ extern "C" {
 
 #define BT_AUDIO_BROADCAST_ID_SIZE               3 /* octets */
 
+/** Life-span states of ASE. Used only by internal APIs
+ *  dealing with setting ASE to proper state depending on operational
+ *  context.
+ *
+ * The values are defined by the ASCS spec and shall not be changed.
+ */
```

Comment:
```suggestion
/** Endpoint states */
```

The comment is no longer valid, and should be updated